### DISCUSSION TITLE
akita vs elf

### DISCUSSION THREAD

**Discussion by Bernitzvah** [salesforce/akita#811] [General]
Hello, I'm developing a demo app to compare akita vs elf. Elf should be more efficent and fater than akita, why is still faster akita? I share with you my src folder from an angular project
[src.zip](https://github.com/datorama/akita/files/8512106/src.zip)

**Comment by NetanelBasal** [comment]
First, I didn't say that Elf is faster than Akita. Can you upload the entire project to Github, please? I'll see what can be done.

**Reply by royling** [reply]
@NetanelBasal btw, will Akita eventually be replaced by Elf? I am comparing the two to see which one to use in a long term...

**Reply by NetanelBasal** [reply]
No, but it'll not evolve anymore. I'd go with Elf.
